dataflow/src/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -47,6 +55,7 @@ public AnalysisResult(
         this.treeLookup = new IdentityHashMap<>(treeLookup);
         this.stores = stores;
         this.finalLocalValues = finalLocalValues;
+        analysisCaches = new IdentityHashMap<>();
```

Comment:
For consistency, write `this.analysisCaches = ...`

Reply:
Note how the three constructors are inconsistent... I would just make each one locally consistent.